pkg/i18n/en_base_config_descriptions.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package i18n
+
+//revive:disable
+var (
+	ConfigLang          = FFM("lang", "Default language for translation (API calls may support language override using headers)")
```

Comment:
Should this be `config.lang`?

Reply:
Yes... not immediately sure why the doc generator didn't complain - but I've fixed it.